composer.json

```
@@ -2,7 +2,7 @@
     "name": "thg/thg-client",
     "description": "THG Open API client",
     "type": "library",
-    "version": "1.3.6",
+    "version": "1.4.0",
```

Depending on which PR is merged earlier (refer https://github.com/UK2group/thghosting-cilent/pull/30), you should update the version again
```suggestion
    "version": "1.4.1",
```